custom_components/alexa_media/const.py

```diff
@@ -15,7 +15,7 @@
     PERCENTAGE,
 )
 
-__version__ = "4.13.4"
+__version__ = "4.13.5"
```

Comment:
just for confirmation @alandtse does it apply also to `const.py` here?

Reply:
Yes. The version is handled by ci. Any mismatch will break it.

Reply:
i take its managed by [this action](https://github.com/alandtse/alexa_media_player/blob/dev/.github/workflows/semantic_release.yml#L79-L80)